pbpy/pbgit.py

```diff
@@ -60,42 +84,43 @@ def check_remote_connection():
 
     current_url = pbtools.run_with_output(["git", "remote", "get-url", "origin"]).stdout
     out = subprocess.run(["git", "ls-remote", "--exit-code", "-h"]).returncode
-    return out == 0, str(current_url)
+    return out == 0, current_url
 
 
 def check_credentials():
-    output = str(subprocess.getoutput("git config user.name"))
+    output = pbtools.run_with_output(["git", "config", "user.name"]).stdout
     if output == "" or output is None:
         user_name = input("Please enter your GitHub username: ")
-        subprocess.call(["git config", "user.name", user_name])
+        subprocess.run(["git config", "user.name", user_name])
 
-    output = str(subprocess.getoutput("git config user.email"))
+    output = pbtools.run_with_output(["git", "config", "user.email"]).stdout
     if output == "" or output is None:
         user_mail = input("Please enter your GitHub email: ")
-        subprocess.call(["git", "config", "user.email", user_mail])
+        subprocess.run(["git", "config", "user.email", user_mail])
 
 
 def sync_file(file_path):
     sync_head = f"origin/{get_current_branch_name()}"
-    return subprocess.call(["git", "restore", "-qWSs", sync_head, "--", file_path])
+    return subprocess.run(["git", "restore", "-qWSs", sync_head, "--", file_path]).returncode
 
 
 def abort_all():
     # Abort everything
-    out = subprocess.getoutput("git merge --abort")
-    out = subprocess.getoutput("git rebase --abort")
-    out = subprocess.getoutput("git am --abort")
-    out = subprocess.getoutput("rm -rf .git/rebase-apply")
+    subprocess.run(["git", "merge", "--abort"])
+    subprocess.run(["git", "rebase", "--abort"])
+    subprocess.run(["git", "am", "--abort"])
```

Comment:
Is run command printing messages to console? If so, please do not redirect output of those commands to console, they all mostly printing errors on a normal pbsync run, because of no merge or rebase going on.